# Add a Makefile to compile according to instructions
# in https://github.com/zgompert/BGC-Bayesian-genomic-clines/blob/main/bgc_manual.pdf
# Added by steven.vandenbrande@kuleuven.be
--- /dev/null	1970-01-01 01:00:00.000000000 +0100
+++ BGC-Bayesian-genomic-clines/Makefile	2023-05-25 19:16:20.145112000 +0200
@@ -0,0 +1,14 @@
+CXX = h5c++
+CXXFLAGS = -Wall -O3
+LIBS = -lgsl -lgslcblas
+
+all: bgc estpost
+
+bgc: 
+	$(CXX) $(CXXFLAGS) -o $@ bgc_main_v1.03.C bgc_func_hdf5_v1.03.C \
+		bgc_func_linkage_v1.03.C bgc_func_ngs_v1.03.C bgc_func_write_v1.03.C \
+		bgc_func_initialize_v1.03.C bgc_func_mcmc_v1.03.C bgc_func_readdata_v1.03.C \
+		mvrandist_v1.03.c $(LIBS)
+
+estpost:
+	$(CXX) $(CXXFLAGS) -o $@ estpost_h5_v1.03.c $(LIBS)
